app/ui.js

```diff
@@ -860,6 +890,18 @@ var UI;
             return val;
         },
 
+        disableSetting: function(name) {
+            var ctrl = document.getElementById('noVNC_setting_' + name);
```

Comment:
A comment about why we have this helper would probably be nice. I.e. it compensates for lack of selectors in CSS.